### PR TITLE
Homework7

### PR DESCRIPTION
**Pull request by biogxh**

### CODE REVIEW

File: Homework-06-documentation-tests.ipynb [hunk 1]
```diff
@@ -57,6 +57,25 @@
...
+    "\n",
+    "class TestStringMethods(unittest.TestCase):\n",
+    "    def test_init(self):\n",
+    "        s = \"one one one two two three\"\n",
+    "        test = TextStats(s)\n",
+    "        \n",
+    "    def count_words(self):\n",
+    "        self.assertEqual(test.count_words,{'one':3,'two':2,'three':1})\n",
+    "\n",
+    "unittest.main()"
```
Comment (jdamerow) [inline Homework-06-documentation-tests.ipynb:76]: you would also write test cases for other situations such as the string is empty and there are commas and periods in the text to make sure your method behaves the way you want it to in all possible scenarios.

File: folderdoc/homework06.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    "authors": Authorinput,
+    "itemType": Typeinput,
+    }
+    temp = datadt['items'] 
+    with open('C:\\Users\\biogx\\GXH\\HW-05-data.json', 'w') as json_file:
+        temp.append(User_entry)
+        datadt['items'] = temp
+        json.dump(datadt, json_file)
+    if i == 0:
+        i = int(input('Press 0 to add a new entry Press 1 to exit'))
```
Comment (jdamerow) [inline folderdoc/homework06.md:33]: there is a specific way to markup code so that it is rendered as code on the page.

**Review by jdamerow — CHANGES_REQUESTED** [review]
